cypress.json

```diff
@@ -1,9 +1,9 @@
 {
     "chromeWebSecurit": false,
     "watchForFileChanges": false,
-    "baseUrl" : "",
+    "baseUrl" : "https://jala3-dev-ed.my.salesforce.com/",
     "env" : {
-        "username" : "",
-        "password" : ""
+        "username" : "edsonarios@freeorg01.com",
```

Comment:
Should be removed your credentials

Reply:
The .env file must be implemented, but it will be implemented in the next iteration